### PR TITLE
Hw 23 1

### PR DESCRIPTION
**Pull request by semyonich**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -60,4 +76,4 @@
             </plugins>
         </pluginManagement>
     </build>
-</project>
+</project>
```
Comment (vicire) [inline pom.xml]: add empty line

File: src/main/webapp/web.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <servlet-name>index</servlet-name>
+        <servlet-class>core.basesyntax.controllers.IndexController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+
+</web-app>
```
Comment (vicire) [inline src/main/webapp/web.xml]: add empty line

**Review by vicire — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (semyonich) [reply pom.xml]: done

File: src/main/webapp/web.xml (hunk 2 above)
Comment (semyonich) [reply src/main/webapp/web.xml]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**